src/components/AppNavigation/AppointmentConfigList.vue

```diff
@@ -106,6 +106,17 @@ export default {
 
 			return !!principal.emailAddress
 		},
+		sortedConfigs() {
+			return [...this.configs].sort((config1, config2) => {
+				if (config1.name < config2.name) {
+					return -1
+				}
+				if (config1.name > config2.name) {
+					return 1
+				}
+				return 0
```

Comment:
I would propose to use `localCompare` here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare
```suggestion
				return config1.name.localCompare(config2.name)
```

Reply:
I knew something like that existed :D